app/forms/DateUtils.scala

```diff
@@ -31,6 +33,14 @@ object DateUtils {
     def formatDate: String = FORMAT.format(date)
 
     def formatDateFull: String = FORMAT_FULL.format(date)
```

Comment:
Seems  function 'formatDateFull' is  redundant can we not remove it.

Reply:
Done, I also removed the formatDate method and associated tests as it was not used in any templates either (and if it was it wouldn't be using Welsh months...)